pkg/kubectl/cmd/create_clusterrolebinding.go

```diff
@@ -54,9 +54,9 @@ func NewCmdCreateClusterRoleBinding(f cmdutil.Factory, cmdOut io.Writer) *cobra.
 	cmdutil.AddPrinterFlags(cmd)
 	cmdutil.AddGeneratorFlags(cmd, cmdutil.ClusterRoleBindingV1GeneratorName)
 	cmd.Flags().String("clusterrole", "", i18n.T("ClusterRole this ClusterRoleBinding should reference"))
-	cmd.Flags().StringSlice("user", []string{}, "usernames to bind to the role")
-	cmd.Flags().StringSlice("group", []string{}, "groups to bind to the role")
-	cmd.Flags().StringSlice("serviceaccount", []string{}, "service accounts to bind to the role")
+	cmd.Flags().StringArray("user", []string{}, "usernames to bind to the role")
+	cmd.Flags().StringArray("group", []string{}, "groups to bind to the role")
+	cmd.Flags().StringArray("serviceaccount", []string{}, "service accounts to bind to the role")
```

Comment:
while, you're touching this, add the expected format to the flag help (`service accounts to bind to the role, in the format <namespace>:<name>`)